addonscript/api.py

```diff
@@ -58,12 +61,18 @@
     upload_status = await get_upload_status(context, locale, pk)
 
     if len(upload_status['files']) != 1:
-        raise SignatureError('Expected 1 file. Got: {}'.format(upload_status))
+        raise SignatureError(
```

Comment:
I'm not entirely sure about the name. Is it really a signature error?

Reply:
This is a bit tricky with the API as it stands now. I've seen in earlier local testing on stage, a langpack failing this with files array empty but no errors logged, because of website metadata being unset for the addon.

So, it is an error (since it was unable to sign), but yea we can rework these exception names if its clearer, though I'm going to defer that renaming at the moment. Johan came up with the current name here anyway.